### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by mnordstr** [shawntan/icentia-ecg#3]
Hi,

Is there an example somewhere on how to use the trained model to analyze a new wfdb record to get beat types as output?

Regards,
Mattias

**Comment by ieee8023** [comment]
No, we converted everything to wfdb after we trained the models. This notebook shows how to open the wfdb data: https://github.com/shawntan/icentia-ecg/blob/master/physionet/wfdb_data_demo.ipynb

**Comment by siddharth-noccarc** [comment]
hi , can i know which patient numbers have afib specifically

**Comment by ieee8023** [comment]
I believe one way to get this is to load each patient and check if the ann.aux_note contains "(AFIB" annotations.
